README.md

```diff
@@ -26,7 +26,13 @@ def create_changeset(params \\ %{}), do: changeset(__MODULE__, params)
 
 #### Actions
 This module takes a schema and filter paramters and runs them through CommonFilters, esentially a wrapper
-around Repo. For more info on filter options take a look at Common Filters
+around Repo. All actions can accept an optional argument of a keyword list that can be used to configure which Repo the Action should use.
+
+## Options
+    * `:repo` - A module that uses the Ecto.Repo Module.
+    * `:replica` - If you don't want to perform any reads against your Primary, you can specify a replica to read from.
```

Comment:
I think we should just have `repo` since replicas would need to be defined via a `repo` anyway, it seems a bit weird to be able to set both when it resolves to only one of the values